explorer/main.go

```diff
@@ -72,16 +71,18 @@ func getBasePath(h http.Header) string {
 func serve(w http.ResponseWriter, r *http.Request) {
 	user, pass, err := GetCredentials()
 	if err != nil {
-		log.Printf("Error getting credentials from KV store: %v", err)
-		w.WriteHeader(http.StatusInternalServerError)
+		fmt.Fprintf(w, "KV explorer credentials not configured.\n")
+		// Log more informative error in cloud logs
+		fmt.Printf("Error getting KV explorer credentials from config store: %v\n", err)
```

Comment:
I believe this line will be printed twice. Once within the `GetCredentials` function and the bubbled-up error handling here.

Reply:
I wasn't sure about this one. I see the second line as a `log`, but logging doesn't work right now for Go, so instead i used a print. The first i see as additional context to the error. If we just do one, I'd prefer the second, since sometime the error is captured by the front end and never displayed in cloud logs.

Reply:
I was trying to point at line 76 and then line 239. I also just noticed that the error message would be of the form:

```
Error getting KV explorer credentials from config store: cannot get credentials pair from config store: {some_err}
```

Reply:
Thanks! Removed the line to avoid the error duplication